services/ui-src/src/libs/formLib.tsx

```diff
@@ -26,6 +26,7 @@ export type OneMACFormConfig = {
   addlInfoTitle?: string;
   addlInfoText?: string | React.ReactNode;
   addlInfoRequired?: boolean;
+  noText?: boolean;
```

Comment:
not sure i like this name givven that all other fields related to additional information section have the addlInfo prefix.....but really who cares since this is all going away *shrug*

Reply:
Yeah, I struggled with the name, for some reason couldn't bring myself to say addlInfo when the label/fieldname for the one to remove is "Change Reason."  I ended up saying noText because it was short and was going to use a better name later and well... it's short at least?